app/assets/stylesheets/logixRelease.scss

```diff
@@ -226,6 +226,14 @@ a.btn.btn-outline-danger {
   text-decoration: none;
 }
 
+.scroll {
+  height: 180px;
+  margin-right: 4px;
+  overflow-y: auto;
+  padding: 4px;
+  text-align: justify;
```

Comment:
text align: left , might be better?

Reply:
![WhatsApp Image 2020-03-20 at 20 21 05](https://user-images.githubusercontent.com/47236824/77175422-a582ef80-6ae8-11ea-8b78-e14d63fc19ce.jpeg)